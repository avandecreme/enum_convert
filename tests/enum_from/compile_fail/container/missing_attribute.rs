use enum_convert::EnumFrom;

enum Source {
    Unit,
}

#[derive(EnumFrom)]
// Missing #[enum_from(Source)] attribute
enum Target {
    #[enum_from]
    Unit,
}

fn main() {}
