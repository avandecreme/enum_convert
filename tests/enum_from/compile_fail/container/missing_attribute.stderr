error: enum_from attribute with source enum names is required
 --> tests/enum_from/compile_fail/container/missing_attribute.rs:7:10
  |
7 | #[derive(EnumFrom)]
  |          ^^^^^^^^
  |
  = note: this error originates in the derive macro `EnumFrom` (in Nightly builds, run with -Z macro-backtrace for more info)
